gensrc/thrift/BackendService.thrift

```diff
@@ -51,6 +51,15 @@ struct TKafkaLoadInfo {
     4: optional map<string, string> properties;
 }
 
+struct TPulsarLoadInfo {
+    1: required string service_url;
+    2: required string topic;
+    3: required string subscription;
+    4: required list<string> partitions;
```

Comment:
I think you can use the same structure with the `TKafkaLoadInfo`

Reply:
The data structs are really different from Kafka and Pulsar:
1. Kafka don't have `subscription` concept;
2. The type of partition for Kafka is `int`, but it's `string` for Pulsar;
3. The type of offset for Kafka is `int`, but it's a structure(MessageID, not used for now) for Pulsar.